src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -57,9 +60,18 @@ public class DatasetService implements ConsentLogger {
   private final UserDAO userDAO;
 
   @Inject
-  public DatasetService(DatasetDAO dataSetDAO, DaaDAO daaDAO, DacDAO dacDAO, ElasticSearchService
-      elasticSearchService, EmailService emailService, OntologyService ontologyService, StudyDAO
-      studyDAO, DatasetServiceDAO datasetServiceDAO, UserDAO userDAO) {
+  public DatasetService(
```

Comment:
Most of the changes from here to my next comment below are largely just formatting changes.

This specific change adjusts the constructor for the new dependency.